# Comment layout test since they aren't installed by default (prevents
# configtest.php from failing).

--- syweb/htdocs/syweb/class_lexer.inc.orig	2007-04-30 21:04:45.004493630 +0100
+++ syweb/htdocs/syweb/class_lexer.inc	2007-04-30 21:04:59.267317747 +0100
@@ -245,7 +245,7 @@
     function _test() {
 	global $symon;
 
-	$this->load('hifn_test.layout');
+	/* $this->load('hifn_test.layout'); */
 	while(!$this->eof) {
 	    $token = $this->next_token();
 	}
